cmd/app/options/options.go

```diff
@@ -184,7 +184,7 @@ func (o *Options) addServerFlags(fs *pflag.FlagSet) {
 		"Address to serve certificates gRPC service.")
 
 	fs.DurationVarP(&o.Server.MaximumClientCertificateDuration,
-		"max-client-certificate-duration", "m", time.Hour*24,
+		"max-client-certificate-duration", "m", time.Hour,
```

Comment:
Suggestion: perhaps link that NIST doc/general reasoning here in a comment.

Reply:
Yep, sounds good to me.